pages/common/xml-canonic.md

```diff
@@ -0,0 +1,20 @@
+# xml-canonic
+
+> Make XML documents canonical.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Make an XML document canonical, including comments (default):
```

Comment:
Perhaps this will read better?
```suggestion
- Make an XML document canonical, including comments (default behaviour):
```

Also, does this mean that it also makes the comments canonical, or that it does not delete them? If it's the latter, then this will need some further rewording to aid clarity here I think.

Reply:
Resolved two places:
xml-canonic.md:- Make an XML document canonical, preserving comments (default behavior):
xml-validate.md:- Validate XML document(s) for well-formedness only (default behavior):